reposcan/test/test_rpm.py

```diff
@@ -0,0 +1,35 @@
+"""
+Tests for rpm packages name parsing.
+"""
+import pytest
+import common.rpm as rpm
+
+
+
+class TestRpmParsing:
+    """ Class for test group for RPM parsing. """
+    test_rpms = []
+
+    @pytest.fixture(autouse=True)
+    def setup_dummy_rpm(self):
+        """ Fixture sets up some RPM packages names for tests. """
+        self.test_rpms.append("fedora-release-workstation-30-1.noarch.rpm")
+        self.test_rpms.append("redhat-internal-NetworkManager-openvpn-profiles-0.1-30.el7.csb.noarch")
+
+    def test_rpmname_parse1(self):
+        """ Test for parsing RPM package name number 1. """
+        name, epoch, version, release, arch = rpm.parse_rpm_name(self.test_rpms[0])
+        assert name == "fedora-release-workstation"
+        assert epoch is None
+        assert version == "30"
+        assert release == "1"
+        assert arch == "noarch"
+
+    def test_rpmname_parse2(self):
+        """ Test for parsing RPM package name number 2. """
+        name, epoch, version, release, arch = rpm.parse_rpm_name(self.test_rpms[1])
```

Comment:
This method `rpm.parse_rpm_name` belongs to `common` package. It already contains tests for this method. So it seems this file is useless :disappointed: .